api/spec/v1.yaml

```diff
@@ -875,15 +907,30 @@ components:
     Transaction:
       type: object
       properties: 
-        height:
+        block:
           type: integer
           format: int64
           description: The block height at which this transaction was executed.
           example: *block_height_1
+        index_in_block:
+          type: integer
+          format: int32
+          description: 0-based index of this transaction in its block
+          example: *block_height_1
+        timestamp:
+          type: string
+          format: date-time
+          # TODO REVIEW: Is this the time the block was _proposed_, or agreed on, or some other time?
```

Comment:
consensus or agreed time?

Reply:
Are you suggesting a different field name? I had that discussion with @pro-wh once on a related timestamp field (I was advocating for a rename), and the conclusion was to leave it as `timestamp` to be consistent with oasis-core.

Reply:
https://github.com/tendermint/tendermint/blob/v0.34.x/spec/core/data_structures.md#header

> weighted median of validators present in the last commit

Reply:
> Are you suggesting a different field name? 

No. I was giving a stab at inferring the correct description, but @pro-wh gave it conclusively.

Reply:
Thanks both. I had little doubt that it was consensus time; not much else makes sense in a distributed trustless setting. But with tendermint's two-round approach (in my understanding, one round to agree that a block was proposed, another round to agree on the block's validity and results), I was not 100% sure which of the rounds counts. Warren's link says it's the former, IIUC. Updated.